docs/tutorials/AddRemark.md

```diff
@@ -23,9 +23,9 @@ For now, let’s keep `RemarkCommand` as simple as possible and print some outpu
 **`RemarkCommand.java`:**
 
 ``` java
-package seedu.address.logic.commands;
```

Comment:
Actl do we need to edit these .md files also? Cause I didn't really touch them since they're AB3's tutorial right, so will still have addressbook and person stuff existing

Reply:
I think ok to edit

Reply:
Ok will do the editing later